sdk/settings/defender.go

```diff
@@ -0,0 +1,58 @@
+package settings
+
+import (
+	"encoding/json"
+
+	"github.com/Hivebrite/twistlock-go/sdk"
+)
+
+type DefenderSettings struct {
+	DisconnectPeriodDays          int    `json:"disconnectPeriodDays"`
+	ListeningPort                 int    `json:"listeningPort"`
+	AutomaticUpgrade              bool   `json:"automaticUpgrade"`
+	AdmissionControlEnabled       bool   `json:"admissionControlEnabled"`
+	AdmissionControlWebhookSuffix string `json:"admissionControlWebhookSuffix"`
+	HostCustomComplianceEnabled   bool   `json:"hostCustomComplianceEnabled"`
+}
+
+func GetDefenderSettings(c sdk.Client) (*DefenderSettings, error) {
+	var unpacker interface{}
+	var marshaled []byte
+
+	req, err := c.NewRequest("GET", "settings/system", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	settings := DefenderSettings{}
+
+	_, err = c.Do(req, &unpacker)
+	if err != nil {
+		return nil, err
+	}
+
+	marshaled, err = json.Marshal(unpacker.(map[string]interface{})["defenderSettings"])
```

Comment:
this one is the only "ugly thing" I did; yet I don't know how to make it better.

there is no endpoint to fetch only "defenderSettings" params.
so I just retrieve the value, then use a unmarshal/marshal to convert it to my defenderSetting struct.